mods/d2k/rules/defaults.yaml

```diff
@@ -241,6 +241,12 @@
 			Radius: 16
 	MapEditorData:
 		Categories: Vehicle
+	GrantConditionOnDamageState@HEAVY:
+		Condition: heavy-damage
+		ValidDamageStates: Heavy
```

Comment:
I think we should remove this line. The defaults are `Heavy` and `Critical` already. Defined like this heavily damaged units are slowed down, but units in the red are not!